### PR TITLE
Add new aliases

### PR DESCRIPTION
**Pull request by AustinXII**
Added a few aliases requested by multiple people for multiple tiers.

### CODE REVIEW

File: data/aliases.js [hunk 1]
```diff
@@ -253,15 +254,21 @@ exports.BattleAliases = {
...
+	"coba": "Cobalion",
 	"cofag": "Cofagrigus",
+	"conk": "Conkeldurr",
 	"cress": "Cresselia",
 	"cube": "Kyurem-Black",
 	"cune": "Suicune",
 	"dnite": "Dragonite",
 	"don": "Groudon",
 	"dogars": "Koffing",
+	"drill": "Excadrill",
```
Comment (TheImmortal) [inline data/aliases.js]: I don't like this because it could mean Beedrill, Drilbur, or even Drill Run.

Comment (ascriptmaster) [inline data/aliases.js]: Seconded. Remove this.

Comment (AustinXII) [inline data/aliases.js]: I don't really mind doing this, but I still think that drill is used more than exca for excadrill

Comment (Slayer95) [inline data/aliases.js]: Never seen ``drill`` for Exca.

Comment (TheImmortal) [inline data/aliases.js]: `mole` is even more common.

File: data/aliases.js [hunk 2]
```diff
@@ -253,15 +254,21 @@ exports.BattleAliases = {
...
 	"ekiller": "Arceus",
 	"esca": "Escavalier",
 	"ferro": "Ferrothorn",
```
Comment (ajadanje) [inline data/aliases.js]: ``"gatr": "Feraligatr",``

**Comment by TheImmortal** [comment]
Are you updating this @AustinXII?

**Comment by AustinXII** [comment]
@TheImmortal yes I will when I get back, should be sometime tomorrow.

**Comment by ascriptmaster** [comment]
Squash the commits. I don't think the build failure is caused by your changes so when you push the squashed changes the test should pass.

**Comment by AustinXII** [comment]
ok @ascriptmaster I squashed

**Pull request merged**